sources/engine/Stride.Engine/Engine/Prefab.cs

```diff
@@ -30,7 +31,33 @@ public sealed class Prefab
         public List<Entity> Instantiate()
         {
             var newPrefab = EntityCloner.Clone(this);
+            CreateNewIds(newPrefab.Entities);
```

Comment:
Shouldn't this be something for `EntityCloner` to do? Seems more like a responsability for it, as its goal is to clone a hierarchy of `Entity`s and `EntityComponent`s so they can be used afterwards.
Thoughts?

Reply:
No, you don't want to duplicate everything. Otherwise, you are never going to be able to reuse stuff.

Reply:
I didn't mean to duplicate it, but to **move** it to `EntityCloner`, so it can clone an entity tree with or without unique ids.